### PR TITLE
Card redesign

### PR DESCRIPTION
**Pull request by aginzberg**

### CODE REVIEW

File: static/css/_base.scss [hunk 1]
```diff
@@ -160,4 +160,12 @@ md-dialog.centered-dialog{
   .active {
     background-color: #E3F2FD;
   }
+}
+span.inset {
+  padding:5px;
+}
+
+.no-surround {
+  margin: 0px !important;
```
Comment (dmorina) [inline static/css/_base.scss]: it's highly recommended not to use !important unless you really have to

Comment (mbernst) [inline static/css/_base.scss]: An alternative to !important is to pay attention to the CSS import order,
or use selectors that are higher precedence (e.g., more selective).

Comment (dmorina) [inline static/css/_base.scss]: yep +1

Comment (aginzberg) [inline static/css/_base.scss]: sure thing. removed important just now.

**Comment by dmorina** [comment]
lgtm

**Pull request merged**